dis_snek/models/discord/channel.py

```diff
@@ -1757,11 +1759,15 @@ async def edit(
 
 @define()
 class GuildPublicThread(ThreadChannel):
```

Comment:
Another fun time where I find an issue.

The `ThreadChannel` object is able to intake a `use_nested_fields` boolean value as a query-string parameter. This requires modification of how forum channels are not only created, but introduces a new breaking change to the behaviour of how data is returned from the API and shown itself. Adding this field into the object will immediately change this PR to a breaking change, FYI.

Reply:
For clarification, this query input dictates whether all messages are nested on the top-level of the `ThreadChannel` schema or not. This is **very breaking**.

Reply:
Alright, I'll ask for someone else's help then with this.

Reply:
This is out-of-scope of this pr.